README.md

```diff
@@ -6,157 +6,90 @@
 
 TiKV has the following key features:
 
-- **Geo-Replication**
+- **Geo-Replication**: TiKV uses [Raft](http://raft.github.io/) and the Placement Driver to support Geo-Replication.
+- **Horizontal scalability**: With PD and carefully designed Raft groups, TiKV excels in horizontal scalability and can easily scale to 100+ TBs of data.
+- **Consistent distributed transactions**: Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions.
+- **Coprocessor support**: Similar to HBase, TiKV implements a coprocessor framework to support distributed computing.
+- **Cooperates with [TiDB](https://github.com/pingcap/tidb)**: Thanks to the internal optimization, TiKV and TiDB can work together to be a compelling database solution with high horizontal scalability, externally-consistent transactions, support for RDBMS, and NoSQL design patterns.
 
-    TiKV uses [Raft](http://raft.github.io/) and the Placement Driver to support Geo-Replication.
+![cncf_logo](images/cncf.png)
 
-- **Horizontal scalability**
+TiKV is a graduated project of the [Cloud Native Computing Foundation](https://cncf.io/) (CNCF). If you are an organization that wants to help shape the evolution of technologies that are container-packaged, dynamically-scheduled and microservices-oriented, consider joining the CNCF. For details about who's involved and how TiKV plays a role, read the CNCF [announcement](https://www.cncf.io/announcements/2020/09/02/cloud-native-computing-foundation-announces-tikv-graduation/).
 
-    With PD and carefully designed Raft groups, TiKV excels in horizontal scalability and can easily scale to 100+ TBs of data.
+## TiKV Adopters
 
-- **Consistent distributed transactions**
+You can view the list of [TiKV Adopters](https://tikv.org/adopters/).
 
-    Similar to Google's Spanner, TiKV supports externally-consistent distributed transactions.
+## Quick Start
 
-- **Coprocessor support**
+The most quickest to try out TiKV is using TiUP:
 
-    Similar to HBase, TiKV implements a coprocessor framework to support distributed computing.
+```shell
+# install tiup
+curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
 
-- **Cooperates with [TiDB](https://github.com/pingcap/tidb)**
+# deploy a TiKV playground
+tiup playground --mode tikv-slim
+```
 
-    Thanks to the internal optimization, TiKV and TiDB can work together to be a compelling database solution with high horizontal scalability, externally-consistent transactions, support for RDBMS, and NoSQL design patterns.
+You can see [TiKV in 5 Minutes](https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/) for a step by step tutorial.
```

Comment:
```suggestion
Watch [TiKV in 5 Minutes](https://tikv.org/docs/5.1/concepts/tikv-in-5-minutes/) for a step-by-step tutorial.
```

Reply:
it's a document, maybe `see` is more suitable?